.github/workflows/checkstyle.yml

```
@@ -0,0 +1,32 @@
+          distribution: 'adopt'
+          
+      - name: Cache Maven packages
+        uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-maven
+          
+      - name: Install Dependencies
+        run: mvn install -DskipTests
+        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)